qualtran/bloqs/qubitization/qubitization_walk_operator.py

```diff
@@ -84,66 +89,71 @@ class QubitizationWalkOperator(GateWithRegisters, SpecializedSingleQubitControll
         Babbush et. al. (2018). Figure 1.
     """
 
-    select: SelectOracle
-    prepare: PrepareOracle
+    block_encoding: BlockEncoding
```

Comment:
One problem here is that the `selection_registers`, `target_registers`, and `junk_registers` methods have been removed from `BlockEncoding` by #1246.

They're still present in `LCUBlockEncoding` and `LCUBlockEncodingZeroState`. Though not optimal, would it make sense for this attribute to be made a `Union[LCUBlockEncoding, LCUBlockEncodingZeroState]` for now so that the attributes can still be used?

Reply:
Sure.